apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.V4Signer.cs

```diff
@@ -170,7 +170,7 @@ void AddHeaders(SortedDictionary<string, string> canonicalized, IDictionary<stri
                                 continue;
                             }
                             var headerName = pair.Key.ToLowerInvariant();
-                            if (headerName == EncryptionKey.KeyHeader || headerName == EncryptionKey.KeyHashHeader || headerName == EncryptionKey.AlgorithmHeader)
+                            if (headerName == EncryptionKey.KeyHeader || headerName == EncryptionKey.KeyHashHeader)
```

Comment:
I'm expecting this code to also be removed given today's meeting.

Reply:
Done.